modules/checking-mco-node-status.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="checking-mco-node-status_{context}"]
+= Checking machine config update status
+
+During upgrades you might want to monitor the progress of individual nodes in the case that issues arise and you need to troubleshoot a node.
+
+To see the status of the Machine Config Operator (MCO) updates to your cluster, use the following `oc` commands:
+
+:FeatureName: Improved MCO state reporting
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+. Get a summary of update statuses for all nodes in a specific machine config pool by running the following command:
++
+[source,terminal]
+----
+$ oc get machineconfignodes
+----
++
+.Example output
+[source,text]
+----
+NAME                          UPDATED   UPDATEPREPARED   UPDATEEXECUTED   UPDATEPOSTACTIONCOMPLETED   UPDATECOMPLETED   RESUMED
+ip-10-0-12-194.ec2.internal   True      False             False              False                    False              False
+ip-10-0-17-102.ec2.internal   False     True              False              False                    False              False
+ip-10-0-2-232.ec2.internal    False     False             True               False                    False              False
+ip-10-0-59-251.ec2.internal   False     False             False              True                     False              False
+ip-10-0-59-56.ec2.internal    False     False             False              False                    True               True
+ip-10-0-6-214.ec2.internal    False     False             Unknown            False                    False              False
+----
++
+where:
++
+UPDATED:: The `True` status indicates that the MCO has applied the current machine config to that particular node. The `False` status indicates that the node is currently updating.
+UPDATEPREPARED:: The `False` status indicates that the MCO has not started reconciling the new machine configs to be distributed. The `True` status indicates that the MCO has completed this phase of the update.
+UPDATEEXECUTED:: The `False` status indicates that the MCO has not started cordoning and draining the node. It also indicates that the disk state and operating system have not started updating. The `True` status indicates that the MCO has completed this phase of the update.
+UPDATEPOSTACTIONCOMPLETED:: The `False` status indicates that the MCO has not started rebooting the node or closing the daemon. The `True` status indicates that the MCO has completed reboot and updating the node status. The `Unknown` status indicates either that an error has occurred during the update process at this phase, or that the MCO is currently applying the update.
```

Comment:
not only this phase has `Unknown` status. every phase has this status, it means the operation is processing

Reply:
updated